frontend/ui/uimanager.lua

```diff
@@ -1082,9 +1073,7 @@ function UIManager:sendEvent(event)
 
     -- Ensure discardEvents
     if self._discard_events_till then
-        local now = { ffiUtil.gettime() }
-        local now_us = now[1] * MILLION + now[2]
-        if now_us < self._discard_events_till then
+        if TimeVal:now() < self._discard_events_till then
```

Comment:
Not the cached value UIManager:getTime() for this one ?
(I guess I'll probably often wonder and ask you the question :)

Reply:
I wasn't *quite* sure it wa snecessary, but I don't have an easy way to test it, because it requires a slow dictionary or something? When I tried to double-check, I found that very few codepaths were using that, and none that I could easily trigger.

So it fell in the "when in doubt" basket, which uses getTime for the *beginning* of a time lapse, and :now for the *end* ;).